engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -427,7 +443,13 @@ def announce_base_product_hybrid(mode)
             params['system_token'] = @system.system_token.presence || params.fetch('system_token', '')
             Rails.logger.info 'No system token' if params['system_token'].blank?
 
-            SccProxy.announce_system_scc("Token token=#{params[:token]}", params, params['system_token'])
+            # TODO: Should this announce_system request include the system's
```

Comment:
@jesusbv @felixsch @digitaltom @rjschwei This one case I wasn't sure about and would appreciated some feedback on...